geomstats/stiefel.py

```diff
@@ -0,0 +1,198 @@
+"""
+Stiefel manifold St(p,n),
+a set of all orthonormal p-frames in n-dimensional space,
+where p <= n
+"""
+
+import logging
+import random
+
+import geomstats.backend as gs
+
+from geomstats.embedded_manifold import EmbeddedManifold
+from geomstats.riemannian_metric import RiemannianMetric
+
+TOLERANCE = 1e-6
+EPSILON = 1e-6
+
+class Stiefel(EmbeddedManifold):
+
+    def __init__(self, p, n):
```

Comment:
Order: n, p is more common for Stiefel manifolds. This is also the order you use L 35.

Reply:
Fixed the order